# DAA-CIBB25
Data and R code used in Avalos et al. A Differential Abundance Analysis Framework for the Indoor Microbiome in Asthma. In proceedings of Computational Intelligence Methods for Bioinformatics and Biostatistics, CIBB 2025. LNBI Springer (Submitted). Genoscreen authorized the release of these OTU tables and metadata from the research project Kamyre


# R Code for Statistical Analysis of the Indoor Microbiome in Asthma Study

## Overview

This repository contains the **R code used to perform the statistical analyses** described in the following conference paper:

> Avalos M., Barrera J., Meza C., Eyheramendy S., Enaud R., *et al.*  
> **Indoor Microbiome in Asthma: A Differential Abundance Framework Accounting for Zero-Inflation, Compositionality, Spatial Clustering, and Covariate Adjustment**  
> *CIBB 2025 – 20th Conference on Computational Intelligence Methods for Bioinformatics and Biostatistics*,  
> Data Science for Bioinformatics Lab, Politecnico di Milano,  
> Milan, Italy, September 2025.  
> HAL reference: https://hal.science/hal-05377271v1

The analyses presented in this repository also support a **journal article currently under submission** to the *Lecture Notes in Bioinformatics (LNBI), Springer* proceedings associated with CIBB 2025.

---

## Purpose of the Repository

The goal of this repository is to:

- Ensure **reproducibility and transparency** of the statistical analyses,
- Allow **free reuse of the code for academic and research purposes**,
- Facilitate methodological extensions and comparisons,
- Promote **proper attribution** to the original data owners (Genoscreen) and authors.

---

## Data Description and Ownership

The datasets analyzed by this code are **not distributed in this repository**.

The data consist of:

- OTU abundance tables for bacterial and fungal communities i.e. targeted metagenomic datasets based on:
  - **16S rRNA gene sequencing (V3–V4 regions)** for bacteria
  - **ITS2 sequencing** for fungi
- Associated metadata
